source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/date-range/index.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+
+import { withTranslations, i18nProps } from '@lib/i18n';
+
+import {
+  formatDate,
+  parseDate,
+} from 'react-day-picker/moment';
+
+import 'react-day-picker/lib/style.css';
+
+function tomorrow() {
```

Comment:
I believe we have this in a lib/time.ts or we should. Also take into account timezones when doing new Date

Reply:
I see we already have _a_ implementation of tomorrow. I'll update that one with this code, and use the `@lib/date`